### DISCUSSION TITLE
MHIG patch

### DISCUSSION THREAD

**Discussion by Chelios107** [Mr-MIBonk/M.I.B._More-Incredible-Bash#23] [Q&A] [answered]
Hi,
Can you help me to find patch or new firmware for my MHIG MMI please?

I have right now:
MHIG_EU_AU_S1544
MU 0397

My FEC is 0210001c and so far I understand that I need other one 023000ee to use 2021/2022 maps.
This comes with patch which I can't find.

Thanks in advanced

**Comment by harman-f** [comment]
MHIG is an older Harman unit version. We only support MHI2 here.

**Reply by Chelios107** [reply]
Okey.
Do you know which version is latest for MHIG?
And where is possible to find latest patch?

**Comment by harman-f** [comment]
Check mibsolution.one

**Comment by Chelios107** [comment]
I did.
No firmware or patch for MHIG

**Comment by harman-f** [comment, marked as answer]
https://mibsolution.one/#/1/9/_pre%20MIB2/MHIG%20-%20HARMAN

**Reply by Chelios107** [reply]
Yes, thank you!
I couldn't find it there.

Can you explain me how to solve this "My FEC is 0210001c and so far I understand that I need other one 023000ee to use 2021/2022 maps." ? How can I update/change FEC only?
That's my goal, I don't need to update firmware I guess.

**Reply by ozchenga** [reply]
> https://mibsolution.one/#/1/9/_pre%20MIB2/MHIG%20-%20HARMAN

Hi harman-f, 

Is there somewhere I can access MHIG_RW_AU for Australian market?
I did a factory reset without knowing my nav/maps would be deleted. Want to update firmware and maps.
Thanks

**Reply by harman-f** [reply]
MHIG is different hardware. Software has many similarities to MHI2, but is not same.
MHI2 AUstralia units do run the same FW as EU units, just different coding.

Maybe this is similar für MHIG - but I do not know
